src/modules/blockchain/implementation/web3-service.js

```diff
@@ -871,6 +871,21 @@ class Web3Service {
         return Number(assertionChunksNumber);
     }
 
+    async getAssertionData(assertionId) {
+        const assertionData = await this.callContractFunction(
```

Comment:
Did you test what is the result of this call?

Reply:
yes. looked like this : [
  BigNumber { _hex: '0x659ff971', _isBigNumber: true },
  BigNumber { _hex: '0x012b', _isBigNumber: true },
  4,
  BigNumber { _hex: '0x04', _isBigNumber: true },
  timestamp: BigNumber { _hex: '0x659ff971', _isBigNumber: true },
  size: BigNumber { _hex: '0x012b', _isBigNumber: true },
  triplesNumber: 4,
  chunksNumber: BigNumber { _hex: '0x04', _isBigNumber: true }
]